src/spaces/discrete.jl

```diff
@@ -117,6 +117,21 @@ function random_empty(model::ABM{<:DiscreteSpace}, cutoff = 0.998)
     end
 end
 
+"""
+    empty_nearby_positions(position, model::ABM{<:DiscreteSpace}, r = 1; kwargs)
+    empty_nearby_positions(agent, model::ABM{<:DiscreteSpace}, r = 1; kwargs)
+
+Return an iterable of all empty positions within "radius" `r` of the given `position` or `agent`.
+
+The value of r and possible keywords operate identically to `nearby_positions`.
+
+This function only exists for discrete spaces with a finite amount of positions.
+"""
+empty_nearby_positions(agent::A, model, r = 1) where {A<:AbstractAgent} = 
```

Comment:
Please expand this to a full `function end` definition and probably the next one too. It's a nice one liner, but if it's on two lines then it's nicer to have it in the full function definition syntax.